server/testing/src/main/java/org/apache/james/util/docker/JamesContainer.java

```diff
@@ -35,34 +34,33 @@
 import org.testcontainers.containers.GenericContainer;
 import org.testcontainers.containers.wait.strategy.WaitStrategy;
 import org.testcontainers.images.builder.ImageFromDockerfile;
-import org.testcontainers.shaded.com.google.common.collect.ImmutableList;
 
-import com.github.dockerjava.api.command.InspectContainerResponse;
-import com.google.common.base.Strings;
-
-public class SwarmGenericContainer implements TestRule {
-    private static final Logger LOGGER = LoggerFactory.getLogger(SwarmGenericContainer.class);
-    private static final String DOCKER_CONTAINER = "DOCKER_CONTAINER";
+public class JamesContainer implements TestRule {
```

Comment:
Bad Idea, how are you going to call the class the put James into a Container ? (: 
What about TestContainerRule ?